_articles/finding-users.md

```diff
@@ -58,7 +58,7 @@ If you don't wish to set up these channels for your project yet, promote your ow
   </p>
 </aside>
 
-**Consider creating a website for your project.** A website makes your project friendlier and easier to navigate, especially paired with clear documentation and tutorials. It also suggests that your project is active, which will make your audience feel more comfortable using it. Use examples to give people ideas for how to use your project.
+**Consider creating a website for your project.** A website makes your project friendlier and easier to navigate, especially when paired with clear documentation and tutorials. A website also suggests that your project is active which will make your audience feel more comfortable using it. Provide examples to give people ideas for how to use your project.
```

Comment:
Can we find another way to start the second sentence?

Reply:
"Supporting a website also suggests ..." ?

Reply:
I'd go with "Having a website also suggests ..."